Xeus does not build without GTEST

Building xeus master from sources does not work without GTEST

```
cmake -D CMAKE_INSTALL_PREFIX=$CONDA_PREFIX ..
```
outputs:
```
-- xeus version: v0.23.1
-- xeus binary version: v1.0.0
-- Performing Test HAS_LTO_FLAG
-- Performing Test HAS_LTO_FLAG - Success
-- CMAKE_CXX_FLAGS: 
-- CMAKE_CXX_FLAGS: 
-- Forcing tests build type to Release
CMake Error at /home/martin/miniconda3/envs/xeus-python/share/cmake-3.15/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR
  GTEST_MAIN_LIBRARY)
Call Stack (most recent call first):
  /home/martin/miniconda3/envs/xeus-python/share/cmake-3.15/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  /home/martin/miniconda3/envs/xeus-python/share/cmake-3.15/Modules/FindGTest.cmake:197 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  test/CMakeLists.txt:79 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/martin/github/martinRenou/xeus/build/CMakeFiles/CMakeOutput.log".
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -299,8 +299,9 @@ endif ()
 # Tests
 # =====
 
+option(BUILD_TESTING "Build tests" OFF)
+option(DOWNLOAD_GTEST "build gtest from downloaded sources" OFF)
 include(CTest)
-OPTION(DOWNLOAD_GTEST "build gtest from downloaded sources" OFF)
 
 if(DOWNLOAD_GTEST OR GTEST_SRC_DIR)
     set(BUILD_TESTING ON)

